### ISSUE TITLE
BOT INIT FAILED Could not login with your username and password... Try again.

### ISSUE THREAD

**Issue by giladfuchs** [hobbydevs/InstabotJS#4]
also many people complain about it on youtube comment.

I try to run it locally on Ubuntu . and I sign in to instagram on my chrome browser.

**Comment by kh-3379** [comment]
plz answer @hobbydevs

- [ BOT INIT FAILED Could not login with your username and password... Try again.]

**Comment by joserealdev** [comment]
I've been doing some research and looks like ig has changed the way you send credentials. Now is encoded instead of plain text

**Comment by cjconnor24** [comment]
Yeah I'm also seeing this issue. @joserealdev any idea what the encoding needs to be?

**Comment by joserealdev** [comment]
@cjconnor24 To be honest, no idea. But if you want to use it anyway, you can just login in the web and get your password encoded, then put it in the config file and change the key password to enc_password in the login form at http.service.js

**Comment by cjconnor24** [comment]
> @cjconnor24 To be honest, no idea. But if you want to use it anyway, you can just login in the web and get your password encoded, then put it in the config file and change the key password to enc_password in the login form at http.service.js

Good call, thanks 👍

**Comment by R1D3** [comment]
how can we get encoded password from web please ?

**Comment by joserealdev** [comment]
> how can we get encoded password from web please ?

Just login and get the password encoded from the ajax call

**Comment by ardasnturk** [comment]
in /services/http.service.js replace the password part 
password: credentials.password,
with the following:  
enc_password: '#PWD_INSTAGRAM_BROWSER:0:{time}:'+credentials.password,
Insta changed the login methods

**Comment by jakeau** [comment]
@ArdaSenturk is this still working I have had no luck with it. Cheers 

Edit:

Magically tried again 30 minutes later and it worked cheers.

**Comment by ghost** [comment]
Can verify what @ArdaSenturk said.

`/services/http.service.js`
```js
const formdata = this.getFormData({
    username: credentials.username,
    enc_password: '#PWD_INSTAGRAM_BROWSER:0:{time}:' + credentials.password,
});
```

**Comment by inquietto** [comment]
How to get de encode password from web? I don't found the method with ajax
